jscripts/post.js

```diff
@@ -181,7 +181,7 @@ var Post = {
 
 		if (totalSize > php_max_upload_size && php_max_upload_size > 0)
 		{
-			var php_max_upload_size_pretty = Math.round(php_max_upload_size / 1e4) / 1e2;
+			var php_max_upload_size_pretty = php_max_upload_size;
```

Comment:
We definitely need a friendly size for this, I can write one if you open this PR to allow modifications from team members - see here for instructions: https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
The `Allow edits from maintainers. ` option is already ticked. Should I change any other settings?

Now working on the `no subject` language entries.

Reply:
Great, I'll work on a function to format file sizes for JS as part of this PR later on today.

Reply:
Borrowed a function from https://stackoverflow.com/a/20732091/6681141